### PR TITLE
Add presubmit job for CAPA ClusterClass test

### PR DESCRIPTION
**Pull request by sedefsavas**
This job will make it possible to run ClusterClass tests on PRs.

### CODE REVIEW

**Comment by sedefsavas** [comment]
cc @richardcase

File: config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-presubmits-clusterclass.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      max_concurrency: 1
+      labels:
+        preset-dind-enabled: "true"
+        preset-kind-volume-mounts: "true"
+        preset-service-account: "true"
+        preset-aws-ssh: "true"
+        preset-aws-credential: "true"
+      spec:
+        containers:
+          - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220708-6b0cfd300e-master
```
Comment (richardcase) [inline config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-presubmits-clusterclass.yaml:22]:
```suggestion
          - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220727-f055b40439-1.24
```

**Review by richardcase — COMMENTED** [review]
Sorry @sedefsavas i missed this.

**Pull request merged**